### ISSUE TITLE
Worldcoin - For every human.

### ISSUE THREAD

**Issue by UjwalKandi** [UjwalKandi/Issue-Bookmarks#399]
**TL;DR** - 
**World ID **
A more human internet with global proof‑of‑personhood. Privacy‑First. Self‑custodial. Decentralized. World ID is a digital passport that lets you prove you are a unique and real person while remaining anonymous.

Use Sign in with Worldcoin to authenticate into web, mobile, and decentralized applications, and privately share your personhood verifications including a phone number for ease, or biometrics for the maximum level of assurance.

**World App**
Simple tools for the age of AI with easy access to decentralized identity and finance.
More human access to the global economy with identity and finance for all. Privacy‑first. No fees. 24-hour support.

It leverages the Worldcoin and Ethereum protocols to help everyone easily authenticate their Proof of Personhood with [World ID](https://worldcoin.org/world-id), send money to anyone for free, claim Worldcoin Grants in countries it’s available*, and explore crypto tokens.

**Worldcoin**
Worldcoin aims to establish universal access to the global economy regardless of country or background. It is designed to become the world's largest human identity and financial network, giving ownership to everyone. All with the intention of welcoming every person on the planet and establishing a place for all of us to benefit in the age of AI.

**Developer Docs**
Building the world's largest identity and financial network as a public utility, giving ownership to everyone.

**Website:**
https://worldcoin.org/home
https://worldcoin.org/world-id

[World App](https://worldcoin.org/world-app)

https://docs.worldcoin.org
